### PR TITLE
Add color

### PR DESCRIPTION
**Pull request by Satoshi-Sh**
Hi Tyler, thanks for making this application. It's really handy. 

I would like to have a color option on this application. If you like the new feature, feel free to incorporate this. 
Thanks.

### CODE REVIEW

**Comment by tyler-simons** [comment]
Hi @Satoshi-Sh sorry just seeing this now -- if you want to make it work with the new set of requirements, I'll merge it in!

**Comment by Satoshi-Sh** [comment]
@tyler-simons Thanks for checking it. I just copied your new requirements.txt. This works fine locally.

File: bg_remove.py [hunk 1]
```diff
@@ -29,14 +30,50 @@ def fix_image(upload):
     fixed = remove(image)
     col2.write("Fixed Image :wrench:")
     col2.image(fixed)
+    
     st.sidebar.markdown("\n")
-    st.sidebar.download_button("Download fixed image", convert_image(fixed), "fixed.png", "image/png")
+    if my_upload:
```
Comment (tyler-simons) [inline bg_remove.py:35]: This if is unncessary and should be `upload` rather than the global `my_upload`

**Comment by tyler-simons** [comment]
Sorry, @Satoshi-Sh I don't think this improves the app. You're welcome to fork it and launch your own version with the code though. It's fully open-source

**Comment by Satoshi-Sh** [comment]
No problem, @tyler-simons. I just wanted this color function on the website. I will publish this on the Streamlit cloud.

**Pull request closed without merging**